themes/src/main/resources/theme/keycloak.v2/account/src/tsconfig.json

```diff
@@ -8,10 +8,11 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "lib": [ "es2015", "dom", "dom.iterable" ],
-    "noImplicitAny": true,
-    "strictNullChecks": true,
+    "noImplicitAny": false,
+    "strictNullChecks": false,
     "jsx": "react",
-    "suppressImplicitAnyIndexErrors": true
+    "suppressImplicitAnyIndexErrors": true,
+    "skipLibCheck": true
```

Comment:
we want to have these checks

Reply:
@edewit yes, I had these changed back, some things were apparently missed on rebase. I'm in the process of updating.